frontend/src/pages/destructionlist/DestructionListCreate.tsx

```diff
@@ -45,30 +46,39 @@ export type DestructionListCreateContext = {
   zaaktypeChoices: ZaaktypeChoice[];
 };
 
+export const getZakenData = async (
```

Comment:
Can we change in this function the way we're awaiting every single call separately? For performance reasons it'd be best to bulk together the three separate awaits `listReviewers()`, `listZaken(params)` and `listZaakTypeChoices()` with a Promise.All()